packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -120,17 +120,17 @@ class BuildIOSFrameworkCommand extends BuildSubCommand {
 
   FlutterProject _project;
 
-  List<BuildMode> get buildModes {
-    final List<BuildMode> buildModes = <BuildMode>[];
+  List<BuildInfo> get buildInfos {
```

Comment:
Rather than plumb through empty dart defines list, I updated most of build framework to operate on BuildInfo instead of BuildMode

Reply:
FYI @jmagman